odk/odk.py

```diff
@@ -210,13 +210,13 @@ class BabelonTranslationProduct(Product):
     language: str = "en"
     """Language tag (IANA/ISO), e.g 'en', 'fr'."""
     
-    include_not_translated: str = "false"
+    include_not_translated: str = False
```

Comment:
These should be declared as `bool`, not `str`:

```python
include_not_translated: bool = False
```

Reply:
oops. weird it worked without complaints